localization/english/bookmark/bookmark_603_l_english.yml

```diff
@@ -1,11 +1,27 @@
 ﻿l_english:
  bm_the_reign_of_chaos:0 "The Reign of Chaos"
- bm_the_reign_of_chaos_desc:0 "PH"
+ bm_the_reign_of_chaos_desc:0 "Years after the end of the Second War, the Eastern Kingdoms are yet to recover entirely from the ravage it caused to lands and people alike. The Alliance is weakened by political and economical crises. Quel'Thalas and other kingdoms distanced themselves from Lordaeron to pursue their own isolated destinies; remnants of the Horde and rebels are still fighting on, while a mysterious epidemy appears in Lordaeron's Eastweald. Rumors spread around the kingdom of a mad prophet's warning to flee an incoming doom and the necessity to flee to the west. Still, prince Arthas Menethil intends to fight on and safeguard his people and his future realm.\nElsewhere, in the arctic continent of Northrend, contact has been lost with all human settlements and colonies. The powerful local empires of the Nerubians and Drakkari Trolls once so mighty are retreating from a new rising power, the Lich King. This mysterious being raises undead armies and prepares for war against the living. Few can oppose him in the north and he might have plans for the rest of the world, too...\n\nWill the kingdoms of the Alliance stick together to face this new crisis or will they split apart driven by survival or interest? Will the Lich King succeed where the Horde failed and bring doom to Azeroth? Who will emerge the victor from the Reign of Chaos?"
 
 
  #Arthas Menethil
  arthas_menethil_character:0 "Prince Arthas"
- arthas_menethil_character_desc:0 "PH"
+ arthas_menethil_character_desc:0 "Only male heir to King Terenas II of Lordaeron, Arthas has been prepared for his future duty by the best tutor Lordaeron can offer, Lord Uther Lightbringer of the Silver Hand, who trained the young prince as a paladin. Skilled and devoted, Arthas is no king yet, serving his father as commander and vassal, eager to prove himself and deeply devoted to his people. King Terenas is an old, wise, and cautious man, it is up to prince Arthas to lead by example and action in the fights to come.\n\nWill Arthas be the ruler that Lordaeron needs or will his recklessness do more bad than good?"
+
+ #Dar'khan Drathir
+ darkhan_character:0 "Dar'Khan"
+ darkhan_character_desc:0 "A magister, mage-lord of Quel'Thalas, Dar'Khan served his liege, king Anasterian, as well as the Alliance against the Trolls and Orcs during the second war, and he did so with great skills and efficiency. The mage, however, is also an expert on the magic of the Sunwell, Quel'Thalas's main source of power, as well as in the arts of magical buildings and esoteric studies. Despite all his accomplishments, Dar'Khan lacks the recognition he deserves and aspires to more than what his ungrateful peers accept to grant him, yet he knows he is capable to achieve greatness and carve his mark in history. Is Dar'Khan promised to a great destiny or will he be forgotten like so many before him that weren't ambitious enough ."
```

Comment:
```suggestion
 darkhan_character_desc:0 "A magister, mage-lord of Quel'Thalas, Dar'Khan served his liege, king Anasterian, as well as the Alliance against the Trolls and Orcs during the second war, and he did so with great skills and efficiency. The mage, however, is also an expert on the magic of the Sunwell, Quel'Thalas's main source of power, as well as in the arts of magical buildings and esoteric studies. Despite all his accomplishments, Dar'Khan lacks the recognition he deserves and aspires to do more than what his ungrateful peers accept to grant him, yet he knows he is capable to achieve greatness and carve his mark in history. Is Dar'Khan promised a great destiny or will he be forgotten like so many before him that weren't ambitious enough?"
```

Reply:
```suggestion
 darkhan_character_desc:0 "A magister, mage-lord of Quel'Thalas, Dar'Khan served his liege, king Anasterian, as well as the Alliance against the trolls and orcs during the second war, and he did so with great skill and efficiency. The mage, however, is also an expert on the magic of the Sunwell, Quel'Thalas's main source of power, as well as in the arts of magical buildings and esoteric studies. Despite all his accomplishments, Dar'Khan lacks the recognition he deserves and aspires to more than what his ungrateful peers accept to grant him, yet he knows he is capable to achieve greatness and leave his mark on history. Is Dar'Khan promised a great destiny, or will he be forgotten like so many before him, that weren't ambitious enough?"
```

Reply:
We had THIS suggestion (the one I posted), 3 weeks ago already, it was never committed...